### PR TITLE
kubeadm-presubmits: remove EKS canary jobs

### PR DESCRIPTION
**Pull request by neolit123**
These jobs do not work. They start and after ~N seconds they return a Pod error. To be able to merge PRs in k/kubeadm we must pass "/skip" everytime.

We can switch the existing jobs to "eks-prow-build-cluster" once it's clear what's going on. Maybe it's due to the "tolerations" and "nodeSelector" sections.

/kind cleanup

### CODE REVIEW

**Comment by SataQiu** [comment]
/lgtm
/hold

This was previously used to investigate the issue（https://github.com/kubernetes/k8s.io/issues/5473）, but it doesn't seem to have progressed. I'm +1 to remove it.

**Review by xmudrii — APPROVED** [review]
Please feel free to remove those jobs, we are taking some other steps to improve the stability and we don't need those jobs at this time. I'll leave it up to you folks to unhold the PR.

/lgtm

**Comment by neolit123** [comment]
> Please feel free to remove those jobs, we are taking some other steps to improve the stability and we don't need those jobs at this time. I'll leave it up to you folks to unhold the PR.
> 

ack
/hold cancel

**Pull request merged**
